content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -260,6 +255,50 @@ The above update rules apply to regular pod updates, but other pod fields can be
 - **Binding:** The `binding` subresource allows setting the pod's `spec.nodeName` via a `Binding` request.
   This is typically only used by the {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}}.
 
+### Pod generation
+
+- The `metadata.generation` field is unique. It will be automatically set by the
+  system such that new pods have a `metadata.generation` of 1, and every update to
+  mutable fields in the pod's spec will increment the `metadata.generation` by 1.
+
+{{< feature-state feature_gate_name="PodObservedGenerationTracking" >}}
```

Comment:
should we add a note to say that k8s is not expecting external controllers to modify this field in any ways and changing it may lead to unexpected behavior?